.github/workflows/build-test.yml

```diff
@@ -239,8 +239,10 @@ jobs:
           rustup component add rust-src --toolchain nightly-2021-12-22
       - name: Install WASM tools
         run: |
-          sudo npm install -g wasm-opt --unsafe-perm
-          sudo npm install -g wabt
+          cd ffi/diplomat/wasm
+          npm ci
+          echo "$HOME/work/icu4x/icu4x/ffi/diplomat/wasm/node_modules/wasm-opt/bin" >> $GITHUB_PATH
```

Comment:
nit: can you leave a comment here saying that if you want to run these tests locally it's easier to just run `sudo npm install -g wasm-opt --unsafe-perm` and `sudo npm install -g wabt`? Folks often look at the github actions source to know which deps they need.

Reply:
I can do this but first allow me to point out that installation of wasm-opt and wabt locally under ffi/diplomat/wasm/node_moduiles with 'npm-ci' works fine as well. It installs everything according to the package.json file, this consolidation is intended.
One still has to set the PATH variable to ffi/diplomat/wasm/[ wasm-opt | wabt ]/bin (instead of GITHUB_PATH) but I think people will figure this out easily
Let me know if you still prefer the comment.

Reply:
Done.